common/src/main/java/org/apache/rocketmq/common/EpochEntry.java

```diff
@@ -64,10 +65,10 @@ public void setEndOffset(long endOffset) {
     @Override
     public String toString() {
         return "EpochEntry{" +
-            "epoch=" + epoch +
-            ", startOffset=" + startOffset +
-            ", endOffset=" + endOffset +
-            '}';
+                "epoch=" + epoch +
```

Comment:
Can we use JSON library to generate string representation instead of manual concatenation?